### PR TITLE
Fixes for various kickstart tests

### PR DESCRIPTION
**Pull request by M4rtinK**
- fix the Fedora live image test (bigger storage & write SUCCESS to result file)
- make the container test authselect aware
- catch some additional & new log messages in monitor

### CODE REVIEW

File: scripts/launcher/lib/log_handler.py [hunk 1]
```diff
@@ -32,10 +32,12 @@ class VirtualLogRequestHandler(LogRequestHandler):
     simple_tests = LogRequestHandler.simple_tests + [
         "Payload setup error:",
         "Out of memory:",
-        "The following group or module is missing:",
+        "Would you like to ignore this and continue with installation?",
+        "Some packages, groups or modules are broken, the installation will be aborted.",
         "Stream was not specified for a module",
         "The following problem occurred on line",  # kickstart parsing error
         "storage configuration failed:",
+        "Disk Requirements:",  # payload does not fit into available storage
```
Comment (jkonecny12) [inline scripts/launcher/lib/log_handler.py]: This is really generic message. Couldn't this make trouble to us in the future?

Comment (M4rtinK) [reply scripts/launcher/lib/log_handler.py]: IIRC there was no better single line message to match on unfortunately. 
So I suggest to merge this now and then go change the message in Anaconda to something more distinct (like "Can't Satisfy Disk Requirements:"). Then once it bubbles it's way through on Fedora & RHEL we can update it here as well. :)

Comment (M4rtinK) [reply scripts/launcher/lib/log_handler.py]: Actually it looks like the message comes from DNF:
https://github.com/rpm-software-management/dnf/blob/e9f362facbaf999348e56b5f6a7bb4c245a1080a/dnf/base.py#L880

But this seems to be prefixed with "Error Summary", which could be a better match.

**Comment by M4rtinK** [comment]
Found a more specify message for out of storage space errors. :) That clears the remaining issue, so merging now. :)

**Pull request merged**